CITATION.cff

```diff
@@ -0,0 +1,181 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: MESSAGEix
+message: >-
+  Thank you for using the MESSAGEix framework! Please cite
+  the GitHub repository and the MESSAGEix framework manuscript 
+  (https://doi.org/10.1016/j.envsoft.2018.11.012).
+type: software
+authors:
+  - given-names: Paul Natsuo
+    family-names: Kishimoto
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-8578-753X'
+  - given-names: Laura
+    family-names: Wienpahl
+  - given-names: Behnam
+    family-names: Zakeri
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0001-9647-2878'
+  - given-names: Francesco
+    family-names: Iovat
+  - given-names: Oliver
+    family-names: Fricko
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-6835-9883'
+  - given-names: Matthew J.
+    family-names: Gidden
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0003-0687-414X'
+  - given-names: Daniel
+    family-names: Huppmann
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-7729-7389'
+  - given-names: Volker
+    family-names: Krey
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0003-0307-3515'
+  - given-names: Adriano
+    family-names: Vinca
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-3051-178X'
+  - given-names: Gamze
+    family-names: Ünlü
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0003-0080-7122'
+  - given-names: Miguel Poblete
+    family-names: Cazenave
+    affiliation: IIASA
+  - given-names: Jarmo S.
+    family-names: Kikstra
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0001-9405-1228'
+  - given-names: Fridolin
+    family-names: Glatter
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0009-0007-9452-2473'
+  - given-names: Jihoon
+    family-names: Min
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-0020-1174'
+  - given-names: Jan
+    family-names: Steinhauser
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-5989-6855'
+  - given-names: Clara
+    family-names: Orthofer
+    orcid: 'https://orcid.org/0000-0002-7049-7791'
+  - given-names: Thomas
+    family-names: Zipperle
+    orcid: 'https://orcid.org/0000-0003-2512-9508'
+  - given-names: Nikolay
+    family-names: Kushin
+  - given-names: Alessio
+    family-names: Mastrucci
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-5611-7780'
+  - given-names: Michael
+    family-names: Pimmer
+  - given-names: Peter
+    family-names: Kolp
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0003-0122-2839'
+identifiers:
+  - type: url
+    value: 'https://github.com/iiasa/message_ix'
+    description: Main repository of the MESSAGEix framework on Github
+  - type: doi
+    value: 10.5281/zenodo.7006303
```

Comment:
You used the version-specific DOI - however, that DOI is only created AFTER a release, so it would not be possible to have the version-number and version-specific DOI correct at the time of the release.

Better to use the version-independent DOI (always refers to the latest version).
```suggestion
    value: 10.5281/zenodo.4005684
```